Palaso.MSBuildTasks/UnixName.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2014 SIL International
+// This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
+// Parts based on code by MJ Hutchinson http://mjhutchinson.com/journal/2010/01/25/integrating_gtk_application_mac
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+
+namespace Palaso.BuildTasks
+{
+	public class UnixName : Task
+	{
+		public override bool Execute()
```

Comment:
A comment might be helpful that lists the values that can get returned from this task. And a class comment that shows how to use it might also be helpful.